examples/raspberry-pi-pwm.rs

```diff
@@ -0,0 +1,21 @@
+use navigator_rs::{pwm_Channel, Navigator};
+use std::thread::sleep;
+use std::time::Duration;
+
+fn main() {
+    println!("Creating your navigator module!");
+    let mut nav = Navigator::new();
+
+    println!("Setting up your navigator, ahoy!");
+    nav.init();
+
+    nav.set_pwm_value(pwm_Channel::C0, 100);
+
+    let channels = &[pwm_Channel::C1, pwm_Channel::C2, pwm_Channel::C3];
+    nav.set_pwm_value_mult(Some(channels), 100);
```

Comment:
hmm, should it be one value to all or a value to each (passed as a slice of values)?

Reply:
I was thinking if we can make a trait, for a single function set_pwm_value,
Then by it's inputs it run differently? 
Something like, if value type is int keep as it is, if <&[]>, use it.

Reply:
yes it can be done with two traits, then you implement both, and the user chooses to import one or the other, or call it using the fully qualified path as trait::method. 

However, I believe it's much easier to use and maintain if we define two regular methods with different names, just like it would be in any simpler language, as `set_pwm_value_all(channel: &[pwm_Channel], value: u16)` and `set_pwm_value_mult<const N: usize>(channel: &[pwm_Channel; N], values: &[u16; N])`.